potential_function/TargetPose[] target_pose_list